### PR TITLE
feat(hostinfo/connections)

### PR DESCRIPTION
**Pull request by kaustavha**
Port waldo reqd ohai plugin to check remote connections on open listening local ports
https://github.com/rackerlabs/ohai-plugins/blob/master/plugins/connections.rb

### CODE REVIEW

File: hostinfo/connections.lua [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+  -- build a "set" (table keyed by address) of local IP addresses
+  local local_ipv4_addresses, local_ipv6_addresses = {}, {}
+  local sigarCtx = sigar:new()
+  local netifs = sigarCtx:netifs()
+  for i=1,#netifs do
+    local info = netifs[i]:info()
+    if info['address'] then
+      table.insert(local_ipv4_addresses, info['address'])
+    end
+    if info['addres6'] then
```
Comment (rphillips) [inline hostinfo/connections.lua]: typo looks like on address6

Comment (kaustavha) [inline hostinfo/connections.lua]: good catch

**Comment by rphillips** [comment]
This branch needs to be rebased to fix the conflicts.

**Comment by rphillips** [comment]
+1

**Pull request merged**